Client/Application/AppDelegate.swift

```diff
@@ -381,6 +381,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
 
+    private func shutdownProfileIfBackgrounded(application: UIApplication) {
+        // Only shutdown the profile if we are still in the background
+        if application.applicationState == UIApplicationState.Background {
```

Comment:
> `Inactive`
> The app is running in the foreground but is not receiving events. This might happen as a result of an interruption or because the app is transitioning to or from the background.

Perhaps use:

```
guard application.applicationState != UIApplicationState.Foreground else {
    return
}
profile?.shutdown()
```

instead?